.env.example

```
@@ -0,0 +1,29 @@
+# Discord Bot Configuration
+DISCORD_TOKEN=your_discord_bot_token_here
+DISCORD_GUILD_ID=your_guild_id_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DISCORD_GUILD_ID key should go before the DISCORD_TOKEN key

(UnorderedKey)